src/com/lftechnology/assignment/jan7/FileCreation.java

```diff
@@ -0,0 +1,101 @@
+package com.lftechnology.assignment.jan7;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.logging.Logger;
+
+/**
+ * Write a program to create a text file in the path c:\java\abc.txt and check whether that file is exists. Using the command exists(),
+ * isDirectory(), isFile(), getName() and getAbsolutePath().
+ * 
+ * This class creates a file and checks its existence after creation by using command exists,isDirectory,isFile and generates it name and
+ * path.
+ * 
+ * @author bibhushan<bibhushanjoshi@lftechnology.com>
+ * 
+ */
+public class FileCreation {
+	private static final Logger LOGGER = Logger.getLogger(FileCreation.class.getName());
+
+	public static void main(String[] args) {
+		File file = new File("abc.txt");
+		try {
+			file.createNewFile();
+			LOGGER.info("File created");
+		} catch (IOException e) {
+			LOGGER.severe("File cannot be created: " + e);
+		}
+		displayFileReport(file);
+	}
+
+	/**
+	 * This method displays the file status about its existence, whether it is a directory and is it actually a file.
+	 * 
+	 * @param file
```

Comment:
describe what the parameter is. Like "{@link File} which cannot be null" or something like this